faunadb/client_test.go

```diff
@@ -2365,6 +2365,24 @@ func (s *ClientTestSuite) TestEchoAnObjectBack() {
 	s.Require().Equal(map[string]string{"key": "value"}, obj)
 }
 
+//see TestStringifyFaunaValues for more
+func (s *ClientTestSuite) TestStringifyQueryV() {
+	var q f.QueryV
+
+	query1 := f.Query(f.Lambda("X", f.Var("X")))
+	expected1 := `Query(Lambda("X", ObjectV{"var": StringV("X")}))`
```

Comment:
I think it's very strange render `Var("X")` as `ObjectV{"var": StringV("X")}`

Reply:
Yes it is. The fix for this will be addressed in DRV-263